docs/user/README.md

```diff
@@ -363,6 +363,24 @@ spec:
   serverImage: private-docker-registry.example.com/dse-img/dse:5f6e7d8c
 ```
 
+## Configuring a NodePort service
+
+A NodePort service may be requested by setting the following fields:
+
+  networking:
+    nodePort:
+      cql: 30001
+      broadcast: 30002
+ 
+The SSL versions of the ports may be requested:
+
+  networking:
+    nodePort:
+      cqlSql: 30010
+      broadcastSql: 30020
```

Comment:
```suggestion
      cqlSsl: 30010
      broadcastSsl: 30020
```

Reply:
and actually maybe SSL should be in caps, because golang idioms

Reply:
doh!